File: CODES/1) C++/11) MASSIF 2 O'LCHAMLI/17.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
	int m,n,k,b=0,c=1;
	cin>>m>>n;
	cout<<"K = ";
	cin>>k;
	int a[m][n];
	for(int i=0;i<m;i++){
		for(int j=0;j<n;j++){
			cout<<"A["<<i<<"]["<<j<<"]=";
			cin>>a[i][j];
		}
	}
  for(int i=0;i<m;i++){
    for(int j=0;j<n;j++){
      cout<<a[i][j]<<" ";
    }
    cout<<endl;
  }
  cout<<endl<<endl;
  for(int i=0;i<n;i++){
  	cout<<a[k-1][i]<<" ";
  	b+=a[k-1][i];
  	c*=a[k-1][i];
  }
  cout<<endl;
  cout<<"Yig'indi = "<<b<<endl;
  cout<<"Ko'paytma = "<<c<<endl;
  return 0;
}

File: LANGUAGES/3) JAVA SCRIPT/11) MASSIF 2 O'LCHAMLI/17.js
let m = 3, n = 3, k = 2, a = [], qator = ""
for (let i = 0; i < m; i++) {
    a[i] = []
    qator = ""
    for (let j = 0; j < n; j++) {
        a[i][j] = Math.floor(Math.random() * 100)
        qator += a[i][j] + " "
    }
    console.log(qator)
}
let sum = 0, f = 1
for (let i = 0; i < n; i++) {
    sum += a[k - 1][i]
    f *= a[k - 1][i]
}
console.log(k, " - qator  yig'indi = ", sum, " Ko'paytma = ", f)